### ISSUE TITLE
Inline loader configuration does not generate keys according with scopePathMap path definition

### ISSUE THREAD

**Issue by ekselys** [ngneat/transloco-keys-manager#37]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

I'm trying to make the key manager, according with its docs, to generate lazy modules keys under related `xx.json` files.

I'm using nx as monorepository manager and this is the structure:

```
project/
├── apps/
│   └── my-app/
│       ├── src/
│       |   |── app/
│       |   |   └── features/
|       |   |       └── feature-one/
|       |   |           |── i18n/
|       |   |           |   └── en.json
|       |   |           |    
|       |   |           └── feature-one.module.ts
|       |   |
|       |   |── home.component.ts
|       |   └── app.module.ts
|       |    
|       └── assets/   
│           └── i18n/
|               └── en.json
|
├── transloco.config.js
└── etc...
```

This is the content of the `transloco.config.file`:

```
module.exports = {
  langs: ['en'],
  scopePathMap: {
    'feature-one': 'apps/my-app/src/app/features/feature-one/i18n'
  }
};
```

And this the `feature-one.module.ts`:

```
export const loader = ['en'].reduce((acc, lang) => {
  acc[lang] = () => import(`./i18n/${lang}.json`);
  return acc;
}, {});

@NgModule({
  imports: [TranslocoModule],
  providers: [
    {
      provide: TRANSLOCO_SCOPE,
      useValue: {
        scope: 'feature-one',
        loader
      }
    }
  ],
  declarations: [YourComponent],
  exports: [YourComponent]
})
export class FeatureOneModule {}
```

If I try to extract the translation keys all keys will be added to `my-app/assets/i18n` folder.

By the way, i've found this workaround for `feature-one.module.ts` definition:

```
 providers: [
    {
      provide: TRANSLOCO_SCOPE,
      useValue: 'feature-one',              // this will make the translation key manager work
    },
    {
      provide: TRANSLOCO_SCOPE,
      useValue: {
        scope: 'feature-one',
        loader
      }
    }
  ],
```

In this way the key manager separates the keys correctly in folder `my-app/assets/i18n` and in `my-app/src/app/features/feature-one/i18n`.

Also, key separation stops working if i do not use a "static value" for the `TRANSLOCO_SCOPE` like this ( _maybe there's a problem with static evaluation?_ ):

```
 providers: [
    {
      provide: TRANSLOCO_SCOPE,
      useValue: FEATURE_ONE_SCOPE, 
    },
  ],
```

## Expected behavior

Correct key separation according with `scopePathMap` parameter.

## Environment

<pre><code>
- transloco-keys-manager:  2.06

</code></pre>

**Comment by shaharkazaz** [comment]
@ekselys can you please verify that this is resolved in v2.0.7? 😄

**Comment by ekselys** [comment]
Thanks, it works now!
Good job!
